mesheryctl/cmd/vars.go

```diff
@@ -131,3 +132,36 @@ func installprereq() {
 	}
 	log.Info("Prerequisite Docker Compose is installed.")
 }
+
+func isMesheryRunning() bool {
+	if runtime.GOOS == "linux" {
+		ps := exec.Command("docker-compose", "-f", dockerComposeFile, "ps")
+		grep := exec.Command("grep", "meshery")
```

Comment:
@mayankshah1607 good update. I think we're ok to include Darwin here as well. `grep` comes with the OS.

Ultimately, the safest and singularly compatible route is to parse the stream / string in Go.

Reply:
@leecalcote yep. Could you take a look now?
Tried a few things that might work for windows, but as you said, the safest approach is to parse the output during runtime itself. I think it would work universally now :)

Reply:
Thins also look neat now as we no longer deal with piping